src/PointwiseFunctions/AnalyticData/GrMhd/MagnetizedFmDisk.hpp

```diff
@@ -97,7 +97,7 @@ class MagnetizedFmDisk
     using type = size_t;
     static constexpr Options::String help = {
         "Grid Resolution for b-field normalization."};
-    static type default_value() { return 255; }
+    static type suggested_value() { return 255; }
```

Comment:
This value is required to faithfully reproduce the numbers for the Accretion Disk comparison. Changing it would ultimately change the normalization the magnetic field magnitude. Just so you know.